File: Linera's Quest/Linera's Quest/Player.m
//
//  Player.m
//  Linera's Quest
//
//  Created by Jakob Hartman on 3/15/14.
//  Copyright (c) 2014 Jakob Hartman. All rights reserved.
//

#import "Player.h"

@implementation Player
-(id) initWithName:(NSString *) aName andTotalHp:(NSInteger) aTotalHp andHp:(NSInteger) aHp  andStrength:(NSInteger) aStrength andPotions:(NSInteger)aPotions andStamina:(NSInteger) aStamina andDefense:(NSInteger)aDefense andExperience:(NSInteger) aExpereince andScore:(NSInteger) aScore andArmor:(NSString *) aArmor andArmorDefense: (NSInteger) aArmorDefense andWeaponDamage:(NSInteger) aWeaponDamage andWeapon:(NSString *) aWeapon andSpecial:(NSString *) aSpecial{
    if(self = [super init]){
        self.name = aName;
        self.totalHp = aTotalHp;
        self.hp = aHp;
        self.strength = aStrength;
        self.defense = aDefense;
        self.stamina = aStamina;
        self.experience = aExpereince;
        self.score = aScore;
        self.potions = aPotions;
        self.armorDefense = aArmorDefense;
        self.armor = aArmor;
        self.weaponDamage = aWeaponDamage;
        self.weapon = aWeapon;
        self.special = aSpecial;
    }
    return self;
}
@end

File: SwiftAction/SwiftAction/Player.swift
//
//  Player.swift
//  SwiftAction
//
//  Created by Jakob Hartman on 6/9/14.
//  Copyright (c) 2014 BYU-Idaho. All rights reserved.
//

import UIKit

class Player{
    var name: String;
    var hp: Int;
    var level:Int;
    var temphp: Int;
    var armor: Int;
    var weapon: Int;
    var potions: Int;
    var strength: Int;
    var defense: Int;
    var totalExp: Int;
    var score: Int;
    var scmod: Int;
    var wEquiped: String;
    var aEquiped: String;
    var kills:Int;
    
    init(){
        name = "Player";
        temphp = 1;
        hp = 10;
        armor = 1;
        level = 1;
        weapon = 1;
        potions = 2;
        strength = 1;
        defense = 1;
        totalExp = 0;
        score = 0;
        scmod = 0;
        wEquiped = "Dagger";
        aEquiped = "Rags";
        potions = 5;
        kills = 0;
    }
    
}
